resources/views/filament/admin/widgets/discord-preview.blade.php

```diff
@@ -0,0 +1,121 @@
+<x-filament-widgets::widget>
+    @assets
+        <link rel="stylesheet" href="{{ asset('/css/filament/admin/discord-preview.css') }}">
+    @endassets
+    <div class="container mx-auto p-4 bg-gray-800 p-4 rounded-lg shadow-lg w-full max-w-full flex items-start mb-4 sender">
+        <div class="relative" style="width: 44px; min-width: 44px; height: 44px; margin-right: 12px;">
+            @if($avatar = data_get($sender, 'avatar'))
```

Comment:
Sorry about this nitpick, but should be easy find/replace.

There should always be a space after each `if`

```suggestion
            @if ($avatar = data_get($sender, 'avatar'))
```